spec/spec.md

```diff
@@ -86,12 +86,38 @@ Both parties MUST have a `service` block containing the following properties:
   }]
 ```
 
-TODO: Explain routing keys. Each property and normative status:
 - service block must be present
 - `id` MUST contain a unique id
 - `type` MUST be `DIDCommMessaging`
 - `serviceEndpoint` MUST be a resolvable URI
-- `routingKeys` MUST contain valid routing keys (ref directly to DIDComm spec section about routing keys)
+- `routingKeys` MUST contain 0 or more valid routing keys. See next section for details.
```

Comment:
if there is 0 routing keys how does the sender encrypt?

Reply:
I think maybe I've been misunderstanding. I was under the impression that all of the encryption in didcomm is done through routing keys. Is the original message encrypted to a different key in the recipients diddoc? If so, which?